pypeit/spec2dobj.py

```diff
@@ -61,6 +61,7 @@ class Spec2DObj(datamodel.DataContainer):
         'objmodel': dict(otype=np.ndarray, atype=np.floating, desc='2D object model image'),
         'ivarmodel': dict(otype=np.ndarray, atype=np.floating, desc='2D ivar model image'),
         'tilts': dict(otype=np.ndarray, atype=np.floating, desc='2D tilts image'),
+        'scaleimg': dict(otype=np.ndarray, atype=np.floating, desc='2D multiplicative scale image that has been applied to the science image'),
```

Comment:
If we are modifying the datamodel, which I presume is desired/necessary, please increment the `version` value.

Reply:
I actually hadn't thought about this.
@kbwestfall -- what do you think?

Reply:
No, I would just increment the version specifically for the datamodel in question.  I don't see a significant reason to tie the datamodel to the pypeit version.  Actually, the main reason they're different, I guess, is so that we can have fine tuned control on the viable version of the datamodel.  I.e., we want old `spec2dobj` files to fault when they're read with the new code, but not the `arcimage` masters, etc.

Reply:
OK - I'll change this to `1.0.1` when I push me next set of commits. That makes sense to me in hindsight.